cmd/ocitools/validate.go

```diff
@@ -397,6 +398,26 @@ func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
 			msgs = append(msgs, fmt.Sprintf("Minimum memory limit should be larger than memory reservation"))
 		}
 	}
+	if r.Network != nil && hostCheck {
+		var exist bool
+		interfaces, err := net.Interfaces()
+		if err != nil {
+			msgs = append(msgs, err.Error())
+			return
+		}
+		for _, prio := range r.Network.Priorities {
+			exist = false
+			for _, ni := range interfaces {
+				if prio.Name == ni.Name {
+					exist = true
+					break
+				}
+			}
+			if !exist {
+				msgs = append(msgs, fmt.Sprintf("Interface %s does not exist on host", prio.Name))
```

Comment:
On Thu, Aug 25, 2016 at 07:56:06PM -0700, Ma Shimiao wrote:
> OK, I will wait until the situation is clear

With the steps from [1]:

* (a) is settled (opencontainers/runtime-spec#538 was merged today).
* (b) has an in-flight PR (opencontainers/runtime-spec#540, submitted
  just now).

[1]: https://github.com/opencontainers/ocitools/pull/200/files#r76177157